### ISSUE TITLE
Broken/invalid link to standard PWA assets on 'ASP.NET Core Blazor Progressive Web Application (PWA)' page.

### ISSUE THREAD

**Issue by michaelm7456** [dotnet/AspNetCore.Docs#31561]
### Description

[Enter feedback here]
On [ASP.NET Core Blazor Progressive Web Application (PWA)](https://learn.microsoft.com/en-us/aspnet/core/blazor/progressive-web-app?view=aspnetcore-8.0&tabs=visual-studio) page, the link to the [ASP.NET Core GitHub repository](https://github.com/dotnet/aspnetcore/tree/main/src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/wwwroot) for getting standard PWA assets is broken/invalid.
![image](https://github.com/dotnet/AspNetCore.Docs/assets/20442719/1b0e99b1-8cfb-42cf-b657-304b147988cf)

Thanks!

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/blazor/progressive-web-app?view=aspnetcore-8.0&tabs=visual-studio

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/blazor/progressive-web-app.md

### Document ID

6240f626-97d4-ace0-5e34-20e1c3592f86

### Article author

@guardrex

**Comment by guardrex** [comment]
Thanks @michaelm7456 ... Yes, the link was broken when that project template was dropped for the 8.0 release.

I'll update the link(s). Until then, you can get the assets from the `release/7.0` branch of their repo ...

https://github.com/dotnet/aspnetcore/tree/release/7.0/src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/wwwroot

**Comment by guardrex** [comment]
Actually .... 🤔 ... that might not be such a great idea given the churn. Hold on a sec ... let me scare up the 8.0 link for the current assets. *Stand-by ..............................*

**Comment by guardrex** [comment]
Here's the 8.0 link ... go with these assets ..................

https://github.com/dotnet/aspnetcore/tree/release/8.0/src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/wwwroot

**Comment by michaelm7456** [comment]
Brilliant @guardrex! Thanks for the rapid response and resolution, much appreciated! 😄

**Comment by guardrex** [comment]
I think that they changed some other aspects as well. I'm looking now to see what else changed. I hope to have a PR fix for this shortly. I'll ping you on the PR so you can see the updates.

**Comment by guardrex** [comment]
Yeah .... for example in some prior release, they changed the manifest filename. I see ...

```html
<link href="manifest.webmanifest" rel="manifest" />
```

... so they went from `manifest.json` to `manifest.webmanifest`.

This is one of those times when I really wish they'd tell me when they're making these types of changes because it breaks YOU and causes me to have to drop my current morning schedule to put out another 🔥🚒. I'll send them a reminder on this point today.
